pkgs/development/compilers/llvm/git/libcxx/fix-libunwind-runtime.patch

```diff
@@ -0,0 +1,12 @@
+--- a/libcxxabi/CMakeLists.txt	2024-03-16 21:45:39.792565851 -0700
++++ b/libcxxabi/CMakeLists.txt	2024-03-16 21:47:24.464622055 -0700
+@@ -47,9 +47,6 @@
+ option(LIBCXXABI_ENABLE_PEDANTIC "Compile with pedantic enabled." OFF)
+ option(LIBCXXABI_ENABLE_WERROR "Fail and stop if a warning is triggered." OFF)
+ option(LIBCXXABI_USE_LLVM_UNWINDER "Build and use the LLVM unwinder." ON)
+-if (LIBCXXABI_USE_LLVM_UNWINDER AND NOT "libunwind" IN_LIST LLVM_ENABLE_RUNTIMES)
+-  message(FATAL_ERROR "LIBCXXABI_USE_LLVM_UNWINDER is set to ON, but libunwind is not specified in LLVM_ENABLE_RUNTIMES.")
+-endif()
```

Comment:
so the standalone libunwind is no longer supported. shouldn't it now be built along with libcxx and libcxxabi? not doing this with libcxxabi led to issues with libcxx which were non-obvious at built time.

Reply:
Not entirely clear, it builds with this patch and nothing breaks so I guess it's fine. Probably worth asking the LLVM project folks about this.

Reply:
we were doing this for libcxxabi thinking it didn't break anything but it did. the compile succeeded but the flags to link libcxx were getting pulled out of libcxxabi conditionally. so it looked ok but it wasn't. this is what i just fixed in PR https://github.com/NixOS/nixpkgs/pull/292043.